app/controllers/api/v1/orders_controller.rb

```diff
@@ -56,6 +56,7 @@ def show
         render json: serializer.new(@order,
           root: root,
           exclude_code_details: true,
+          include_messages: bool_param(:include_messages, false),
```

Comment:
As mentioned further below, I think we shouldn't send the messages with `orders` payload but instead use the `api/v1/messages` endpoint. If we include messages in orders, then we won't be able to filter out is_private messages and the order creator will receive them.